doc/modules/ROOT/pages/graph-analytics-serverless.adoc

```diff
@@ -223,6 +223,9 @@ memory = sessions.estimate(
     node_count=20,
     relationship_count=50,
     algorithm_categories=[AlgorithmCategory.CENTRALITY, AlgorithmCategory.NODE_EMBEDDING],
+    node_label_count=1,
+    node_property_count=1,
+    relationship_property_count=1
```

Comment:
we should document these with some prose as well. I am thinking that with this many parameters, an actual table would be a good idea

Reply:
thats what I think the API reference is good for. Will add a link :)